lms/static/js/commerce/views/receipt_view.js

```diff
@@ -91,15 +92,43 @@ var edx = edx || {};
                 dataType: 'json'
             }).retry({times: 5, timeout: 2000, statusCodes: [404]});
         },
+        /**
+         * Retrieve credit provider data from LMS.
+         * @param  {object} order The basket that was purchased.
+         * @return {object}                 JQuery Promise.
```

Comment:
Changed it to None because it does not returns any value instead it renders the receipt page.